modules/openapi-generator/src/main/resources/lua/api.mustache

```diff
@@ -24,8 +24,10 @@ local function new_{{classname}}(host, basePath, schemes)
 		schemes_map[v] = v
 	end
 	local default_scheme = schemes_map.https or schemes_map.http
+	local port = http_util.split_authority(host, default_scheme)
```

Comment:
Is the 'host' that is passed into this function expected to be an authority? If so the argument should be renamed

Also, `http_util.split_authority` returns the host *and* the port, the usage here is incorrect.